packages/contracts-bedrock/test/DataAvailabilityChallenge.t.sol

```diff
@@ -0,0 +1,496 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Test } from "forge-std/Test.sol";
+import { DataAvailabilityChallenge, ChallengeStatus, Challenge } from "../src/L1/DataAvailabilityChallenge.sol";
+import { Proxy } from "src/universal/Proxy.sol";
+
+address constant DAC_OWNER = address(1234);
+uint256 constant CHALLENGE_WINDOW = 1000;
+uint256 constant RESOLVE_WINDOW = 1000;
+uint256 constant BOND_SIZE = 1000000000;
+uint256 constant RESOLVER_REFUND_PERCENTAGE = 50;
+
+contract DataAvailabilityChallengeTest is Test {
```

Comment:
It may be worth making this inherit from `CommonTest` instead of `Test` and fixing up `CommonTest` to make sure that it grabs the address of the `dac` after the deploy script deploys the contract. The idea is that for the `L1` contracts, we use the deploy script to set up the state for the tests

Reply:
makes sense, refactored to be a `CommonTest`